src/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Symfony\WebpackEncoreBundle\EventListener;
+
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookup;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupCollection;
+
+class ExceptionListener
+{
+    private $entrypointLookupCollection;
+
+    private $entrypointNames;
+
+    public function __construct(EntrypointLookupCollection $entrypointLookupCollection, array $entrypointNames)
+    {
+        $this->entrypointLookupCollection = $entrypointLookupCollection;
+        $this->entrypointNames = $entrypointNames;
+    }
+
+    public function onKernelException()
+    {
+        foreach ($this->entrypointNames as $entrypointName) {
+            $this->entrypointLookupCollection->getEntrypointLookup($entrypointName)->reset();
+        }
+    }
+}
```

Comment:
Could we add a very basic unit test for this class? There isn't much to test, but having at least ONE basic test will protect us from making any super silly typos now or in the future.